presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueToPrestoConverter.java

```diff
@@ -51,6 +53,16 @@
 
     private GlueToPrestoConverter() {}
 
+    public static <T, R> List<R> mappedCopy(List<T> list, Function<T, R> mapper)
```

Comment:
Make this private and put it at the end of the class (we put private methods at the end)

Reply:
i'd rather see the existing `io.prestosql.util.MoreLists#mappedCopy` being improved & made reusable for hive connector.

Reply:
Currently doesn't seem like there are any good options. The `presto-spi` module intentionally avoids depending on Guava and isn't for utilities anyway, so there doesn't seem to be a good location to share the implementation from.

Reply:
We have some shared classes in `presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/util`
(as the name suggests, this is not an ideal location, so once we have more of them, we can find a better home for them)

Reply:
Having a few duplicated methods between connectors is preferable to `presto-plugin-toolkit` becoming a "junk drawer". There's a reason we don't have a generic `util` or `common` module in Presto. This is fine for now.